README.md

```diff
@@ -19,3 +19,22 @@ API request that resulted in a `401`.
 If your application does not wish to log these actions at all, use
 `\Psr\Log\NullLogger` for any constructors that require a
 `\Psr\Log\LoggerInterface`.
+
+
+## Overview of main classes
+
+### Client
+MPX API implementation of Guzzle ClientInterface. As a Client it doesn’t do anything extra but suppress errors to force a returning HTTP 200.
```

Comment:
Let's link to ClientInterface over in the Guzzle repository?

`suppress errors to force a returning HTTP 200.` => `suppress errors to force a returning HTTP 200 for XML responses.`